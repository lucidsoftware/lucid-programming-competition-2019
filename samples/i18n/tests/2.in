40
wsxewfpyg
t
biexkbrzdbgz
sosfnkdbjp
cobzsgqmi
n
mdyrmpmcpxmkf
fymskhsnxhs
mdxsohogpn
anlp
agitjivroh
lclfbyb
hwm
tpcrksgd
rkkowpvi
ic
fwdba
xurv
k
pe
faaqzazktnztjg
mpktfke
vpsddxzdbehqrc
chhpqkrdyiwu
oqt
byshwx
vryekgjb
e
kyt
nhivpldx
yldp
aw
yrshjistnnkhc
kemtnrxxw
tnins
hitj
ayihoe
mhvsurm
cylq
qrrhbgptogl
