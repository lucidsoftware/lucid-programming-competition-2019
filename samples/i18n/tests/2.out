w7g
t
b10z
s8p
c7i
n
m11f
f9s
m8n
a2p
a8h
l5b
h1m
t6d
r6i
ic
f3a
x2v
k
pe
f12g
m5e
v12c
c10u
o1t
b4x
v6b
e
k1t
n6x
y2p
aw
y11c
k7w
t3s
h2j
a4e
m5m
c2q
q9l
